www/index.html

```diff
@@ -330,15 +330,15 @@
                   data-template="details"
                 >
                   <div class="item-media">
-                    <img width="80" src="{{this.album.images[0].url}}">
+                    <img width="80" src="{{this.artworkUrl100}}">
```

Comment:
Shame iTunes doesn't have a higher res image than this :/

Reply:
Yeah, the options were 30, 60 and 100 :(